client/components/widget/data_viz/gviz/gviz-directive.js

```diff
@@ -271,8 +254,9 @@ explorer.components.widget.data_viz.gviz.gvizChart = function(
           let promise = queryResultDataService.
               fetchResults(scope.widgetConfig);
           $timeout(function() {
-            scope.widgetConfig.state().datasource.status =
-                ResultsDataStatus.FETCHING;
+            // TODO(klausw): make behavior conditional on if queuing is enabled.
```

Comment:
As we're not going to allow opt-out on this, this section seems safely removeable.